pyvista/plotting/mapper.py

```diff
@@ -270,7 +270,7 @@ def scalar_map_mode(self) -> str:
     def scalar_map_mode(self, scalar_mode: Union[str, FieldAssociation]):
         if isinstance(scalar_mode, FieldAssociation):
             scalar_mode = scalar_mode.name
-        scalar_mode = scalar_mode.lower()
+        scalar_mode = scalar_mode.lower()  # type: ignore
```

Comment:
Stop this!!!

Reply:
The madness continues in https://github.com/pyvista/pyvista/pull/3837

Reply:
But really, I'll remove it here.